charts/victoria-metrics-agent/values.yaml

```diff
@@ -0,0 +1,67 @@
+# Default values for victoria-metrics-agent.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: victoriametrics/vmagent
+  tag: "" # rewrites Chart.AppVersion
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+args:
```

Comment:
the basic idea of every chart is to work by default, do you have any idea how this can be achieved?